field_records/ubna_2023.csv

```diff
@@ -108,7 +108,11 @@ Date and Time deployed (local) , Date and Time recovered (local) , AudioMoth # ,
-2023-09-28T10:52:00 , (DATE-TIME) , J , 017 , Telephone Field , 47° 39' 25.080'' N , 122° 17' 29.502'' W , 192000 , Medium , None , None , 4.181 , (VOLTAGE) , AK , CT , (UPLOADER) , (RECOVER-DATE) , Ikea Batteries Nighttime 0:00-15:00 UTC
-2023-09-28T10:52:00 , (DATE-TIME) , I , 004 , Foliage , 47° 39' 18.882'' N , 122° 17' 26.802'' W , 192000 , Medium , None , None , 4.284 , (VOLTAGE) , AK , CT , (UPLOADER) , (RECOVER-DATE) , Panasonic Batteries Nighttime 0:00-15:00 UTC;
-2023-09-28T10:54:00 , (DATE-TIME) , K , 002 , Central Pond , 47° 39' 17.802'' N , 122° 17' 34.140'' W , 192000 , Medium , None , None , 4.177 , (VOLTAGE) , AK , CT , (UPLOADER) , (RECOVER-DATE) , Ikea Batteries Nighttime 0:00-15:00 UTC;
-2023-09-28T10:54:00 , (DATE-TIME) , L , 016 , E18 Bridge , 47° 39' 21.288'' N , 122° 17' 48.720'' W , 192000 , Medium , None , None , 4.173 , (VOLTAGE) , AK , CT , (UPLOADER) , (RECOVER-DATE) , Ikea Batteries Nighttime 0:00-15:00 UTC;
+2023-09-28T10:52:00 , 2023-10-04T10:12:00 , J , 017 , Telephone Field , 47° 39' 25.080'' N , 122° 17' 29.502'' W , 192000 , Medium , None , None , 4.181 , 3.588 , AK , CT , MS , recover-20231004 , Ikea Batteries Nighttime 0:00-15:00 UTC
+2023-09-28T10:52:00 , 2023-10-04T11:01:00 , I , 004 , Foliage , 47° 39' 18.882'' N , 122° 17' 26.802'' W , 192000 , Medium , None , None , 4.284 , 3.640 , AK , CT , MS , recover-20231004 , Panasonic Batteries Nighttime 0:00-15:00 UTC;
+2023-09-28T10:54:00 , 2023-10-04T10:55:00 , K , 002 , Central Pond , 47° 39' 17.802'' N , 122° 17' 34.140'' W , 192000 , Medium , None , None , 4.177 , 3.630 , AK , CT , MS , recover-20231004 , Ikea Batteries Nighttime 0:00-15:00 UTC;
+2023-09-28T10:54:00 , 2023-10-04T10:26:00 , L , 016 , E18 Bridge , 47° 39' 21.288'' N , 122° 17' 48.720'' W , 192000 , Medium , None , None , 4.173 , 3.620 , AK , CT , MS , recover-20231004 , Ikea Batteries Nighttime 0:00-15:00 UTC;
+2023-10-04T09:48:00 , (DATE-TIME) , A , 008 , Foliage , 47° 39' 25.638'' N , 122° 17' 29.442'' W , 192000 , Medium , None , None , 4.250 , (VOLTAGE) , AK , CT , MS , (RECOVER-DATE) , Ikea Batteries Nighttime 1:15-14:30 UTC; First day training Mila + Mollie;
+2023-10-04T09:48:00 , (DATE-TIME) , C , 001 , Central Pond , 47° 39' 17.412'' N , 122° 17' 34.578'' W , 192000 , Medium , None , None , 4.226 , (VOLTAGE) , AK , CT , MS , (RECOVER-DATE) , Ikea Batteries Nighttime 1:15-14:30 UTC; First day training Mila + Mollie;
+2023-10-04T09:47:00 , (DATE-TIME) , D , 009 , Telephone Field , 47° 39' 17.802'' N , 122° 17' 34.140'' W , 192000 , Medium , None , None , 4.289 , (VOLTAGE) , AK , CT , MS , (RECOVER-DATE) , Ikea Batteries Nighttime 1:15-14:30 UTC; First day training Mila + Mollie;
```

Comment:
> 2023-10-04T09:47:00 , (DATE-TIME) , D , 009 , Telephone Field , 47° 39' 17.802'' N , 122° 17' 34.140'' W , 192000 , Medium , None , None , 4.289 , (VOLTAGE) , AK , CT , MS , (RECOVER-DATE) , Ikea Batteries Nighttime 1:15-14:30 UTC; First day training Mila + Mollie;

Hi @FXF48-Mila I think you may have forgot to update the lat/lon details for this row because its the same as Central Pond's row for the previous week,

> 2023-09-28T10:54:00 , 2023-10-04T10:55:00 , K , 002 , Central Pond , 47° 39' 17.802'' N , 122° 17' 34.140'' W , 192000 , Medium , None , None , 4.177 , 3.630 , AK , CT , MS , recover-20231004 , Ikea Batteries Nighttime 0:00-15:00 UTC;

Could you check again for this? Thanks!